sal_interfaces/Dome/Dome_Events.xml

```diff
@@ -1,268 +1,157 @@
-            </item>
-</SALEvent>
-<SALEvent>
-    <Subsystem>Dome</Subsystem>
-    <Version>3.8.42</Version>
-    <Author></Author>
-    <EFDB_Topic>Dome_logevent_rearDoorStatusChange</EFDB_Topic>
-    <Alias>rearDoorStatusChange</Alias>
-    <Explanation>http://sal.lsst.org</Explanation>
-            <item>
-                <EFDB_Name>rearDoorStatusChangeCode</EFDB_Name>
-                <Description>Return the code of the rear door status change</Description>
-                <IDL_Type>int</IDL_Type>
-                <Units>unitless</Units>
-                <Count>1</Count>
-            </item>
-</SALEvent>
-<SALEvent>
-    <Subsystem>Dome</Subsystem>
-    <Version>3.8.42</Version>
-    <Author></Author>
-    <EFDB_Topic>Dome_logevent_sealStatusChange</EFDB_Topic>
-    <Alias>sealStatusChange</Alias>
-    <Explanation>http://sal.lsst.org</Explanation>
-            <item>
-                <EFDB_Name>seatStatusChangeCode</EFDB_Name>
-                <Description>Return the code of the seal status change</Description>
-                <IDL_Type>int</IDL_Type>
-                <Units>unitless</Units>
-                <Count>1</Count>
-            </item>
-</SALEvent>
-<SALEvent>
-    <Subsystem>Dome</Subsystem>
-    <Version>3.8.42</Version>
-    <Author></Author>
-    <EFDB_Topic>Dome_logevent_interlockAlarm</EFDB_Topic>
-    <Alias>interlockAlarm</Alias>
-    <Explanation>http://sal.lsst.org</Explanation>
-            <item>
-                <EFDB_Name>interlockAlarmCode</EFDB_Name>
-                <Description>Return the code of the interlock alarm</Description>
-                <IDL_Type>int</IDL_Type>
-                <Units>unitless</Units>
-                <Count>1</Count>
-            </item>
-</SALEvent>
+    <SALEvent>
+        <Subsystem>Dome</Subsystem>
+        <EFDB_Topic>Dome_logevent_brakesEngaged</EFDB_Topic>
+        <item>
+            <EFDB_Name>brakes</EFDB_Name>
+            <Description>Bitmask of brakes that are engaged.
+            The bits are defined by the Brake enum.</Description>
+            <IDL_Type>int</IDL_Type>
+            <Units>unitless</Units>
+            <Count>1</Count>
+        </item>
+    </SALEvent>
+    <SALEvent>
+        <Subsystem>Dome</Subsystem>
+        <EFDB_Topic>Dome_logevent_lockingPinsEngaged</EFDB_Topic>
+        <item>
+            <EFDB_Name>engaged</EFDB_Name>
+            <Description>Bitmask of locking pins that have been engaged.
+            The bits are defined by the LockingPin enum.</Description>
+            <IDL_Type>int</IDL_Type>
+            <Units>unitless</Units>
+            <Count>1</Count>
+        </item>
+    </SALEvent>
+    <SALEvent>
+        <Subsystem>Dome</Subsystem>
+        <EFDB_Topic>Dome_logevent_interlocks</EFDB_Topic>
+        <item>
+            <EFDB_Name>interlocks</EFDB_Name>
+            <Description>Bitmask of safety interlocks that are latched.
+            The bits are defined by the Interlock enum.</Description>

```

Comment:
Where is this enum defined?

Reply:
Again I'd need the names of the interlocks to make the bitmask.

I guess the question is whether to start with stupid names and then improve them later or just wait until we know more. I lean towards the latter.

Reply:
We haven't discussed this yet. So let's clarify this and then make an update to this event.